packages/ui/Scrollable/Scrollable.js

```diff
@@ -589,6 +613,10 @@ class ScrollableBase extends Component {
 		this.isDragging = true;
 		this.dragStartX = this.scrollLeft + this.getRtlX(ev.x);
 		this.dragStartY = this.scrollTop + ev.y;
+
+		if (this.props.onDragStart) {
+			forward('onDragStart', ev, this.props);
```

Comment:
We _usually_ put the `forward` at the top of the function to ensure we always forward (unless there are occasions in which we stop propagation). Do you know of a reason why we need them at the end?

Reply:
No real reason that I'm aware of. The placement at the end of the handler here was more-or-less consistency with the other events within the file, although I'm not certain those have a specific purpose or could/should be moved to the top either. I can change these, but the other handlers (`onMouseDown`, `onFlick`) could be verified as well later.